<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="jalebi12345" failures="3" tests="15" name="Edge Tests" time="124.71" errors="0" timestamp="2025-06-13T17:41:24 IST">
  <testcase classname="tests.LoginTest" name="testAuthFormInitialState" time="0.84"/>
  <testcase classname="tests.DropDownTest" name="testLoginWithValidCredentials" time="2.911"/>
  <testcase classname="tests.DropDownTest" name="testDropdownMenuIsVisible" time="0.386"/>
  <testcase classname="tests.LoginTest" name="testValidLogin" time="3.38"/>
  <testcase classname="tests.DropDownTest" name="testClickContactLink" time="0.243">
    <failure type="java.lang.AssertionError" message="Should navigate to Contact page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Should navigate to Contact page. expected [true] but found [false]
at tests.DropDownTest.testClickContactLink(DropDownTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
at java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)
at java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:108)
at java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:269)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testClickContactLink -->
  <testcase classname="tests.LoginTest" name="testInvalidLogin" time="2.407"/>
  <testcase classname="tests.DropDownTest" name="testClickChangePassword" time="0.635">
    <failure type="java.lang.AssertionError" message="Should navigate to Change password page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Should navigate to Change password page. expected [true] but found [false]
at tests.DropDownTest.testClickChangePassword(DropDownTest.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
at java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)
at java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:108)
at java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:269)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testClickChangePassword -->
  <testcase classname="tests.LoginTest" name="testModeSwitching" time="2.23"/>
  <testcase classname="tests.DropDownTest" name="testHoverOverDropDownItems" time="0.0">
    <skipped/>
  </testcase> <!-- testHoverOverDropDownItems -->
  <testcase classname="tests.LoginTest" name="testRegistration" time="17.883">
    <failure type="java.lang.AssertionError" message="Registration should be successful with valid data expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Registration should be successful with valid data expected [true] but found [false]
at tests.LoginTest.testRegistration(LoginTest.java:121)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
at java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)
at java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:108)
at java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:269)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testRegistration -->
  <testcase classname="tests.LoginTest" name="testFieldVisibilityByMode" time="1.621"/>
  <testcase classname="tests.LoginTest" name="testAnimationTransitions" time="1.072"/>
  <testcase classname="tests.LoginTest" name="testMethodChaining" time="2.513"/>
  <testcase classname="tests.LoginTest" name="testPasswordMismatchRegistration" time="2.235"/>
  <testcase classname="tests.LoginTest" name="testEmptyFormValidation" time="3.323"/>
</testsuite> <!-- Edge Tests -->
